NOTICE.txt

```diff
@@ -311,219 +311,6 @@ OTHER DEALINGS IN THE SOFTWARE.
 
 
 
----
```

Comment:
this got changed to PW, maybe we need to regenerate the notice again.

Reply:
++ sorry if I'm breaking stuff, trying to figure out why the electron process won't launch for e2e tests.